### PR TITLE
Feature/ws technologies doc

### PR DESCRIPTION
**Pull request by radeklawgmin**
List of technologies applied in WS

## Description
A table with technologies that DS Team has used in WS.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [x] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code standards](/CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

**Comment by radeklawgmin** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -18,6 +18,38 @@ and then see the results on [localhost:8080/apps/websight/index.html/content::sp
...
+
+### CMS
+
+Websight CMS makes use of the following technologies:
+| | | | | | | | |
+|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|
+| <a href="https://www.java.com"><img src="/assets/technologies/java-logo.png" alt="Java" style="width:80px;" /></a> | <a href="https://felix.apache.org"><img src="/assets/technologies/osgi-logo.png" alt="OSGi" style="width:90px;" /></a> | <a href="https://sling.apache.org"><img src="/assets/technologies/sling-logo.png" alt="Apache Sling" style="width:70px;" /></a> | <a href="https://www.jackrabbit.apache.org"><img src="/assets/technologies/apachejackrabbit-logo.png" alt="Apache Jackrabbit" style="width:90px;" /></a> | <a href="https://reactjs.org"><img src="/assets/technologies/react-logo.png" alt="ReactJS" style="width:90px;" /></a> | <a href="https://www.typescriptlang.org"><img src="/assets/technologies/typescript-logo.png" alt="TypeScript" style="width:50px;" /></a> | <a href="https://www.nginx.com"><img src="/assets/technologies/nginx-logo.png" alt="nginx" style="width:80px;" /></a> | <a href="https://www.mongodb.com"><img src="/assets/technologies/mongodb-logo.png" alt="MongoDB" style="width:90px;" /></a>
```
Comment (paweljozwicki) [inline README.md:30]:
```suggestion
| <a href="https://www.java.com"><img src="/assets/technologies/java-logo.png" alt="Java" style="width:80px;" /></a> | <a href="https://felix.apache.org"><img src="/assets/technologies/osgi-logo.png" alt="OSGi" style="width:90px;" /></a> | <a href="https://sling.apache.org"><img src="/assets/technologies/sling-logo.png" alt="Apache Sling" style="width:70px;" /></a> | <a href="https://www.jackrabbit.apache.org"><img src="/assets/technologies/jackrabbit-logo.png" alt="Apache Jackrabbit" style="width:90px;" /></a> | <a href="https://reactjs.org"><img src="/assets/technologies/react-logo.png" alt="ReactJS" style="width:90px;" /></a> | <a href="https://www.typescriptlang.org"><img src="/assets/technologies/typescript-logo.png" alt="TypeScript" style="width:50px;" /></a> | <a href="https://www.nginx.com"><img src="/assets/technologies/nginx-logo.png" alt="nginx" style="width:80px;" /></a> | <a href="https://www.mongodb.com"><img src="/assets/technologies/mongodb-logo.png" alt="MongoDB" style="width:90px;" /></a>
```

Comment (paweljozwicki) [reply README.md:30]: I had broken jackrabbit logo

**Comment by radeklawgmin** [comment]
I have read the Dynamic Solutions Individual Contributor License Agreement (CLA) Document and I hereby sign the CLA

**Comment by radeklawgmin** [comment]
recheck

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**